de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/annotations/AnnotatedDocument.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.annotations;
```

Comment:
@reckart: As of commit 9955b2f8b0b05918d9507fe7ab0ecfacbe2f216b, all the new classes should have the Apache license header from the old classes.